Server/Jobs/RunStackwalkTaskJob.cs

```diff
@@ -101,16 +96,11 @@ public async Task Execute(Guid taskId, CancellationToken cancellationToken)
 
         if (task.DeleteDumpAfterRunning)
         {
-            await semaphore.WaitAsync(cancellationToken);
-            try
+            using (await localTempFileLocks.LockAsync(baseFolder, cancellationToken).ConfigureAwait(false))
```

Comment:
And this as well could be an using statement.

Reply:
Also had to change to `using var asyncKeyedLock = await fileLocks.LockAsync(baseFolder, cancellationToken).ConfigureAwait(false);`

Reply:
Do you still want it this way or shall I revert? It even had to go on two lines.

Reply:
It'd probably fit without the ConfigureAwait, but I think it's fine enough as you seem to like that and used it elsewhere where this is used.

Reply:
OK, so just renamed to baseFolderLock.